openwisp_controller/config/static/config/css/lib/advanced-mode.css

```diff
@@ -950,14 +949,15 @@ table.jsoneditor-search button.jsoneditor-previous:hover {
 .jsoneditor-text-errors td pre{ margin: 0 }
 
 .full-screen{
-  position: absolute;
-  top: 0px;
-  left: 0px;
-  padding-left: 0px;
-  padding-right: 0px;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  padding-left: 0;
+  padding-right: 0;
   background-color: #ffffff;
-  width: 100%;
-  z-index: 9;
+  z-index: 11;
```

Comment:
the z-index of the main menu on the right is 9999, this z-index should be higher than that one or the following can happen:

![why-zindex-needs-to-be-higher](https://user-images.githubusercontent.com/841044/90851036-1e58d280-e339-11ea-8cde-6498f4c641d1.gif)